SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2853,6 +2860,7 @@ void USpatialNetDriver::PostSpawnPlayerController(APlayerController* PlayerContr
 	PlayerController->SetReplicates(true);
 	PlayerController->Role = OriginalRole;
 	PlayerController->SetPlayer(OwnershipConnection);
+	PlayerController->ChangeState(NAME_Playing);
```

Comment:
@sjtsilvester context on this? This will get called when the pawn possesses a pawn, so not sure why we need to manually call it here.

Reply:
Do you think we should ask Intrepid about it?

Reply:
@m-samiec : Looks like we added this line to Intrepid at around the time of 0.13-RC to fix a bug. If the code that calls this when the pawn possesses a pawn was added after 0.13-RC, then I think we can remove this, otherwise I shall need to dig for the exact bug this was causing.